libcontainer/rootfs_linux.go

```diff
@@ -219,6 +219,11 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 			}
 		}
 		return nil
+	case "overlay":
+		if err := mountPropagate(m, rootfs, mountLabel); err != nil {
+			return err
+		}
+		return label.SetFileLabel(m.Destination, mountLabel)
```

Comment:
You should not call this.
The label would have to be specified in the overaly stuff above I believe.